rotkehlchen/db/snapshots.py

```diff
@@ -263,19 +268,26 @@ def _import_snapshot(
         balances_list: List[Dict[str, str]],
         location_data_list: List[Dict[str, str]],
     ) -> Tuple[bool, str]:
+        """Import the validated snapshot data to the database."""
         processed_balances_list = []
         processed_location_data_list = []
 
-        for entry in balances_list:
-            processed_balances_list.append(
-                DBAssetBalance(
-                    category=BalanceType.deserialize(entry['category']),
-                    time=Timestamp(int(entry['timestamp'])),
-                    asset=Asset(identifier=entry['asset_identifier']),
-                    amount=entry['amount'],
-                    usd_value=str(FVal(entry['usd_value'])),
-                ),
-            )
+        identifiers = [entry['asset_identifier'] for entry in balances_list]
```

Comment:
You probably should revert this to how you had it before with only the nft though since now that you don't put any asset inside you would end up with an unknown identifier in the assets table and it would to to the `UnknownAsset` exception handler.

Nothing bad would happen but that identifier would stay forever in the assets table as it would not be detected or deleted anywhere.